src/assets/main.css

```diff
@@ -13,9 +13,9 @@
 .dice-roller > div {
     display: inline-block;
 }
-.dice-roller span {
+/* .dice-roller span {
     font-weight: 700;
-}
+} */
```

Comment:
I'm not sure I like the idea of just blanket removing the bold. Maybe make this an option in settings, or target embedded rollers specifically?

Reply:
You're right!
I did that change based on one example provided in another issue where the output was:

`**Gender**: `dice; [[Note^Table]]``

With the bold coming from the .dice-roller rule, this wasn't looking good (to my eyes at least).

But... it's better to keep the bold by default and let users who want to do something like above to add their own css snippet to remove it.

So... all of this to say: I'll remove this change.